source/administration/config-settings.rst

```diff
@@ -2702,23 +2702,21 @@ Lists installed plugins on your Mattermost server and whether they are enabled.
 Require Plugin Signature
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**True**:
+**True**: Require valid plugin signatures before starting managed or unmanaged plugins. Prepackaged plugins are not subject to plugin signature verification. Pre-packaged plugins are not subject to plugin signature verification. Plugins installed through the Plugin Marketplace are always subject to plugin signature verification at the time of download.
```

Comment:
Looks like `Pre-packaged plugins are not subject to plugin signature verification.` got duplicated twice -- sorry for that!